src/templates/common/src/common/UI/StyledLayout/index.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+
+const StyledLayout = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`
+
+export default StyledLayout
```

Comment:
there's currently a discussion about always using named exports.

what's your opinion @pedrobslisboa & @danilowoz?

Reply:
As I said before, I would prefer to only use named-exports in order to avoid different names for the same components, I know that probably it will not be the intention of the developer, but it might happen by chance. 

But for me, it's no big deal. I think we already took a lot of advantages creating a name convention in the last discussion.

Reply:
well, I'm okay with only using named exports so maybe we should take advantage of this PR to push these conventions?

i.e.: change all default exports to named exports

Reply:
I kind like to use named exported for everything that is not the main component.

Ex:

```js
import Modal, { ModalHeader, ModalContent, ModalFooter } from 'some-random-ui-library' 
```

But I have no problem to use only named exports

Reply:
that's also what I like to do.

but to be honest it requires a bit of knowledge from the developer to identify the "main" export and there are some cases where it can be a bit relative.

I don't know...

Reply:
Ok, I've just changed some imports to fit in this way, agreeing with @danilowoz when he sais:

> I think we already took a lot of advantages creating a name convention in the last discussion.

On the future, with more discussions and ideas we can turn it into a more mature thing